Add slider package to react-components (#21506)
microsoft/fluentui: packages/react-components/src/Alpha/Alpha.stories.mdx

import { Meta } from '@storybook/addon-docs';

<Meta title="Components-Alpha/Overview" />

# Alpha Components

## About alpha controls

> ❗These component are available for limited production use❗
> 
> ❗The APIs will change before final release as functionality is still being added.❗

Reviewer: Maybe?
```suggestion
❗ These component are available for limited production use, but their APIs may change before final release as functionality is still being added. ❗
```
